MobiusCore/Source/EffectHandlers/EffectRouter.swift

```diff
@@ -72,6 +72,12 @@ public struct EffectRouter<Effect, Event> {
     }
 }
 
+extension EffectRouter: _EffectHandlerConvertible {
+    public func _asEffectHandlerConnectable() -> AnyConnectable<Effect, Event> {
```

Comment:
Should the `_asEffectHandlerConnectable()` methods be public?

Reply:
Yes; this is necessary since protocols can’t have multiple levels of access control. In order to publicly conform to the protocol the method must be implemented publicly.